ui/selectors/selectors.js

```diff
@@ -751,7 +751,10 @@ export function getIpfsGateway(state) {
 }
 
 export function getInfuraBlocked(state) {
-  return Boolean(state.metamask.infuraBlocked);
+  return (
+    state.metamask.networksMetadata[getSelectedNetworkClientId(state)]
+      .status === NetworkStatus.Blocked
+  );
```

Comment:
should we add a test for this selector in `selectors.test.js`?

Reply:
added